web/src/components/apps/Dashboard.tsx

```diff
@@ -851,7 +867,15 @@ class Dashboard extends Component<Props, State> {
                       gettingAppLicenseErrMsg={
                         this.state.gettingAppLicenseErrMsg
                       }
-                    />
+                    >
+                      {/* leaving this here as an example: please delete later */}
+                      {/* <LicenseTester
```

Comment:
component to test the useLicenseWithIntercept hook, please delete when it's unnecessary for you.